src/commands/AdventofcodeCommand.ts

```diff
@@ -46,29 +46,78 @@ class AdventOfCodeCommand extends Command {
 				const name = !member.name ? member.id : member.name.padEnd(memberNameLength, " ");
 				const score = member.local_score;
 				const stars = member.stars.toString().padEnd(starLength);
+				const position = (i + 1).toString().padStart(2, " ");
 
-				list = list.concat(`${name} | ${stars} | ${score}\n`);
+				list = list.concat(`${position}) ${name} | ${stars} | ${score}\n`);
 			}
 		}
 
 		return list.concat("```");
 	}
 
-	async run(message: Message): Promise<void> {
-		const amountInTopList = 15;
-		const link = `https://adventofcode.com/${ADVENT_OF_CODE_YEAR}/leaderboard/private/view/${ADVENT_OF_CODE_LEADERBOARD}`;
+	getYear() {
+		const date = new Date();
+
+		if (date.getMonth() > 10) {
+			return date.getFullYear();
+		}
+
+		return date.getFullYear() - 1;
+	}
```

Comment:
I think this makes more sense to live in `utils/DateUtils.ts`, I can see it being reused.

Reply:
How would you suggest calling such a function, I can do "getYearBasedOnMonth", but that will most likely get more questions that answers.

Reply:
Ah, ignore this actually. It is quite specific. I misread the logic.